StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/OnLoadErrorMessageHandler.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  OnLoadErrorMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/9/24.
+//
+
+import Foundation
+
+// Emitted when there is an error loading connect js
+class OnLoadErrorMessageHandler: OnSetterFunctionCalledMessageHandler<OnLoadErrorMessageHandler.Values> {
+    struct Values: Codable, Equatable {
+        let elementTagName: String
```

Comment:
The JS layer is sending `elementTagName` since it's already built into the public API, but we're not going to do anything with it in the mobile layer, so I think we can drop it. Dropping it just enables the web view to change it later without breaking the SDK and causing a payload decoding error.

```suggestion
```